src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -110,7 +110,7 @@ HandlerResolver::LoadRequestHandlerAssembly(const IHttpApplication &pApplication
 
         LOG_INFOF(L"Loading request handler:  '%ls'", handlerDllPath.c_str());
 
-        hRequestHandlerDll = LoadLibrary(handlerDllPath.c_str());
+        hRequestHandlerDll = LoadLibraryEx(handlerDllPath.c_str(), 0, LOAD_LIBRARY_SEARCH_DEFAULT_DIRS | LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR);
```

Comment:
> ##[error]src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\HandlerResolver.cpp(113,0): error C26477: (NETCORE_ENGINEERING_TELEMETRY=Build) Use 'nullptr' rather than 0 or NULL (es.47).